airbyte-ci/connectors/pipelines/pipelines/contexts.py

```diff
@@ -197,16 +206,21 @@ def get_repo_dir(self, subdir: str = ".", exclude: Optional[List[str]] = None, i
         Returns:
             Directory: The selected repo directory.
         """
+        if exclude and include:
+            raise ValueError("exclude and include cannot be used together")
+        default_excluded_files = self.get_default_excluded_files(subdir)
         if exclude is None:
-            exclude = self.DEFAULT_EXCLUDED_FILES
+            exclude = default_excluded_files
         else:
-            exclude += self.DEFAULT_EXCLUDED_FILES
+            exclude += default_excluded_files
             exclude = list(set(exclude))
```

Comment:
Nit: could simplify and remove the if statement by
```py
exclude = exclude or [] # not sure we even need this?
exclude += default_excluded_files
exclude = list(set(exclude))
exclude.sort()
```

Reply:
Fair enough! This should handle the None case for exclude
![image](https://github.com/airbytehq/airbyte/assets/1325608/6d7ee1b4-eef5-4de0-8d87-9e35db7163ab)

But this is a nit!